### PR TITLE
English changelog +support +contribute +upgrade

### PR DESCRIPTION
**Pull request by timse201**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,24 +68,19 @@ Not literally “protect from”. The plugin’s purpose is to _detect_ any “h
...
 
 ### 1.3.5 ###
-* Deutsch: Optimierungen für WordPress 3.8
-* English: Optimizations for WordPress 3.8
+* Optimizations for WordPress 3.8
 
 ### 1.3.4 ###
-* Deutsch: Benachrichtigung per E-Mail, sobald [Google Safe Browsing](http://en.wikipedia.org/wiki/Google_Safe_Browsing) Malware im Blog erkennt. [Mehr auf Google+](https://plus.google.com/110569673423509816572/posts/H72FFwvna1i)
-* English: [Google Safe Browsing](http://en.wikipedia.org/wiki/Google_Safe_Browsing) for malware and phishing monitoring.
+* [Google Safe Browsing](http://en.wikipedia.org/wiki/Google_Safe_Browsing) for malware and phishing monitoring with e-mail notification. [Additional informations on Google+](https://plus.google.com/110569673423509816572/posts/H72FFwvna1i) (only german)
```
Comment (swissspidy) [inline README.md]: Information is always singular in English. Languages like German are always capitalized.

**Comment by timse201** [comment]
shouldnt we remove the compatibility notice?

**Comment by swissspidy** [comment]
Which notice do you mean?  The "WordPress 4.x ready" one or the "Upgrade Notice"?

**Comment by timse201** [comment]
the wp 4.x ready under features


Remove the check for include and require commands (#wpforce)
don't know how to translate und what exactly is meant

**Comment by Zodiac1978** [comment]
There is this last small typo ... german -> German ;)
https://github.com/pluginkollektiv/antivirus/pull/26#discussion-diff-78060606

**Pull request merged**